workspaces/adventure-pack/goodies/python3/src/transpose_matrix/test.py

```diff
@@ -0,0 +1,51 @@
+from . import *
+import pytest
+
+
+def test_improper_input() -> None:
+    with pytest.raises(ValueError):
+        assert len(transpose_matrix([]))
```

Comment:
This should be sufficient to trigger the error, right? And same below.

```suggestion
        transpose_matrix([])
```